modules/planning/tasks/deciders/path_bounds_decider/path_bounds_decider.cc

```diff
@@ -370,23 +390,42 @@ bool PathBoundsDecider::SearchPullOverPosition(
           curr_s, &curr_lane_left_width, &curr_lane_right_width);
       if (std::fabs(curr_right_bound + adc_half_width - curr_lane_right_width) >
           FLAGS_pull_over_road_edge_buffer) {
-        ADEBUG << "Not close enough to lane-edge -> "
-               << "Not feasible for pull-over.";
+        ADEBUG << "Not close enough to lane-edge. Not feasible for pull-over.";
         is_feasible_window = false;
         break;
       }
+
+      // check rightmost driving lane:
+      //   NONE/CITY_DRIVING/BIKING/SIDEWALK/PARKING
```

Comment:
Please fix the indent of the comment

Reply:
kinda on purpose.  but I'll delete this indent 'coz it does look weird.